rcl_yaml_param_parser/README.md

```diff
@@ -0,0 +1,25 @@
+**ROS2 rcl YAML paramter parser**
+
+Parse a YAML parameter file and populate the C data structure(params_st)
+
+The data structure params_st will then be used during node initialization
```

Comment:
params_st is just a variable name used in function definition, I recommend referring to the type here or generally to "a C datastructure"

Reply:
Fixed it